src/state/reducers/month-state/employee-info.reducer.ts

```diff
@@ -24,17 +24,37 @@ export function employeeInfoReducerF(name: string) {
     action: ScheduleActionModel | ActionModel<WorkerInfoExtendedInterface>
```

Comment:
Myślę, że jeśli zmienić typ `action` na ActionModel<WorkerInfoExtendedInterface>, to można pozbyć się tego dużego if'a, który jest niżej (27 - 43 linijki )